pkg/scheduler/watchers/membercluster/controller_integration_test.go

```diff
@@ -150,6 +155,112 @@ var _ = Describe("scheduler member cluster source controller", Serial, Ordered,
 		})
 	})
 
+	Context("ready cluster has a non-resource property change", func() {
+		BeforeAll(func() {
+			Consistently(noKeyEnqueuedActual, consistentlyDuration, consistentlyInterval).Should(Succeed(), "Workqueue is not empty")
+
+			// Retrieve the cluster.
+			memberCluster := &clusterv1beta1.MemberCluster{}
+			Expect(hubClient.Get(ctx, types.NamespacedName{Name: clusterName1}, memberCluster)).To(Succeed(), "Failed to get member cluster")
+
+			// Update the list of non-resource properties.
+			memberCluster.Status.Properties = map[clusterv1beta1.PropertyName]clusterv1beta1.PropertyValue{
+				dummyNonResourcePropertyName: {
+					Value:           dummyNonResourcePropertyValue,
+					ObservationTime: metav1.NewTime(time.Now()),
+				},
+			}
+			Expect(hubClient.Status().Update(ctx, memberCluster)).Should(Succeed(), "Failed to update member cluster non-resource properties")
+		})
+
+		It("should enqueue CRPs (case 1a)", func() {
+			Eventually(qualifiedKeysEnqueuedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Keys are not enqueued as expected")
```

Comment:
```suggestion
			Eventually(qualifiedKeysEnqueuedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Keys are not enqueued")
```
"as expected" sounds like it's a expected behavior.

Reply:
Oh, it could be failing because the keys are not enqueued, or some keys are enqueued, but not the expected set of keys, hence the description.